app/embeds/GoogleDriveVideo.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import GoogleDriveVideo from "./GoogleDriveVideo";
+
+describe("GoogleDriveVideo", () => {
+  const match = GoogleDriveVideo.ENABLED[0];
+  test("to be enabled on share link", () => {
```

Comment:
In my testing a share link has `?usp=sharing` in the query string, it would cause it not to match here – lets make sure it accounts for anything on the query string too